File: app/src/main/java/org/sabda/gpt/AppService.java
package org.sabda.gpt;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

import androidx.annotation.Nullable;

public class AppService extends Service {

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        return START_STICKY;
    }

    @Nullable
    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
}

File: app/src/main/java/org/sabda/gpt/broadcastservice/AppService.kt
package org.sabda.gpt.broadcastservice

import android.app.Service
import android.content.Intent
import android.os.IBinder

class AppService : Service() {
    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        return START_STICKY
    }

    override fun onBind(intent: Intent): IBinder? {
        return null
    }
}
